api/src/test/java/org/openmrs/module/sync2/api/impl/SyncConfigurationServiceImplTest.java

```diff
@@ -3,23 +3,42 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.openmrs.module.sync2.api.SyncConfigurationService;
 import org.openmrs.module.sync2.api.exceptions.SyncException;
 import org.openmrs.module.sync2.api.model.configuration.ClassConfiguration;
 import org.openmrs.module.sync2.api.model.configuration.GeneralConfiguration;
 import org.openmrs.module.sync2.api.model.configuration.SyncConfiguration;
 import org.openmrs.module.sync2.api.model.configuration.SyncMethodConfiguration;
+import org.openmrs.module.sync2.api.scheduler.SyncSchedulerService;
+import org.openmrs.module.sync2.api.scheduler.impl.SyncSchedulerServiceImpl;
 import org.openmrs.module.sync2.api.utils.SyncUtils;
 
 import java.util.Arrays;
 import java.util.List;
 
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
 public class SyncConfigurationServiceImplTest {
 
     private static final String sampleFeedConfigurationPath = "sampleSyncConfiguration.json";
     private static final String sampleFeedConfigurationPath2 = "sampleSyncConfiguration2.json";
 
+    @Mock
+    private SyncSchedulerService syncSchedulerService;
```

Comment:
This field is unused. Is this intentional?

Reply:
Yes, it is needed to mock scheduler service when the saveConfiguration method is checked.